hr_timesheet_predefined_description/models/account_analytic_line.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Tecnativa - Juan José Seguí
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    predefined_description_id = fields.Many2one(
+        comodel_name="timesheet.predefined.description", string="Predefined Description"
+    )
+
+    def write(self, vals):
+        if "predefined_description_id" in vals:
```

Comment:
Only do that if no name is specified:

```suggestion
        if "predefined_description_id" in vals and (vals.get("name") or "/") == "/":
```